### PR TITLE
Reconfig Eslint for test sub-project

### PR DESCRIPTION
**Pull request by aserputov**
## Issue This PR Addresses

Fixes: #3002

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: test/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: '@senecacdot/eslint-config-telescope',
+  overrides: [
+    {
+      files: ['./**/*.js', '*.test.js', '*.test.ts', '*.test.tsx', '**/__mocks__/**/*.js'],
+      plugins: ['react-native'],
```
Comment (JiaHua-Zou) [inline test/.eslintrc.js:6]: I don't think we need the `react-natice` plugin.

**Review by JiaHua-Zou — DISMISSED** [review]
Looks good!

**Review by tpmai22 — DISMISSED** [review]
LGTM !

**Comment by tpmai22** [comment]
@aserputov this option give you a chance to keep the approved reviews and also rebase your branch

**Comment by tpmai22** [comment]
image

**Review by tpmai22 — APPROVED** [review]

**Comment by JiaHua-Zou** [comment]
Just going to rerun the CI to see if the e2e test fails again.

**Pull request merged**